packages/embark/src/cmd/cmd_controller.js

```diff
@@ -441,12 +441,12 @@ class EmbarkController {
   }
 
   ejectWebpack() {
-    var embarkConfig = fs.embarkPath('dist/lib/modules/pipeline/webpack.config.js');
+    var embarkConfig = fs.embarkPath('./node_modules/embark-pipeline/dist/webpack.config.js');
     var dappConfig = fs.dappPath('webpack.config.js');
     fs.copyPreserve(embarkConfig, dappConfig);
     console.log(__('webpack config ejected to:').dim.yellow);
     console.log(`${dappConfig}`.green);
-    var embarkOverrides = fs.embarkPath('dist/lib/modules/pipeline/babel-loader-overrides.js');
+    var embarkOverrides = fs.embarkPath('./node_modules/embark-pipeline/babel-loader-overrides.js');
```

Comment:
These too are very dangerous and I'd like to explore other options for this before merge

Reply:
Actually, can just search for `node_modules/embark-pipeline` and then build the other paths from the found path.